demos/link-prediction/homogeneous-comparison-link-prediction.ipynb

```diff
@@ -0,0 +1,1515 @@
+{
```

Comment:
<p>Could you add a <code>## Conclusion</code> heading to this cell?</p><p> </p><p>It might also be good to link to the demos for the specific algorithms, like:</p><p> </p><pre class="ql-syntax" spellcheck="false">For more information about the link prediction process, all of these algorithms have specific demos with more details:

- [Node2Vec](node2vec-link-prediction.ipynb)
- ...
</pre><p> </p><p>This gives a user some "next" steps to follow if they get to the end of this demo and are interested in further things.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1658/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
<p>I ever tried to give a conclusion. However, due to the randomness of edge train-test split, the results of each run is different. In most cases, Attri2Vec performs best, while the performance ranks of Node2Vec, GraphSAGE and GCN change significantly in different runs. It is hard to draw a consistent conclusion. Do you think it deserves to fix the random seed for splitting the train/test edge sets?</p><p> </p><p> </p><p>I think it is helpful to instruct readers to refer to the link prediction demos for the specific algorithms.</p>

Reply:
> However, due to the randomness of edge train-test split, the results of each run is different. In most cases, Attri2Vec performs best, while the performance ranks of Node2Vec, GraphSAGE and GCN change significantly in different runs. It is hard to draw a consistent conclusion. Do you think it deserves to fix the random seed for splitting the train/test edge sets?

I don't understand? I was just meaning adding the heading to the existing cell without changing the content?

Reply:
<p>Sorry! I misunderstood. I though you were to ask me to add an ## Conclusion section to discuss the experimental results. </p><p> </p><p>It is fine to just add a a&nbsp;<code>## Conclusion</code>&nbsp;heading.</p><p> </p>